task-1/1.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	var a float64
+	var b float64
```

Comment:
переменные несут один смысл, поэтому лучше не разделять их
```suggestion
	var a, b float64
```